src/redux/books/booksSlice.js

```diff
@@ -1,32 +1,59 @@
 import { createSlice } from '@reduxjs/toolkit';
 
 const initialState = {
-  books: [],
+  books: [
+    {
+      item_id: 'item1',
+      title: 'The Great Gatsby',
+      author: 'John Smith',
+      category: 'Fiction',
+      progressPorcentage: 0,
+      currentChapter: 'unknown',
+    },
+    {
+      item_id: 'item2',
+      title: 'Anna Karenina',
+      author: 'Leo Tolstoy',
+      category: 'Fiction',
+      progressPorcentage: 0,
+      currentChapter: 'unknown',
+    },
+    {
+      item_id: 'item3',
+      title: 'The Selfish Gene',
+      author: 'Richard Dawkins',
+      category: 'Nonfiction',
+      progressPorcentage: 0,
+      currentChapter: 'unknown',
+    },
+  ],
 };
 
 const booksSlice = createSlice({
-  name: 'books',
+  name: 'bookshelf',
   initialState,
   reducers: {
     addBook: (state, action) => {
-      const { bookName, author } = action.payload;
+      const { title, author } = action.payload;
 
       const newBook = {
-        bookName,
+        item_id: 'item' + state.books.length,
+        title,
         author,
-        genre: 'unknown',
+        category: 'unknown',
         progressPercentage: 0,
         currentChapter: 'unknown',
       };
 
```

Comment:
**[OPTIONAL]**

- [ ] It would be nice if you make your own logo of your name and put that in the readme file at the top to present yourself as a brand in the job market or tech industry to catch the recruiters' eyes 👍